glue/core/command.py

```diff
@@ -119,7 +122,9 @@ def do(self, cmd):
         """
         Execute and log a new command
 
-        :rtype: The return value of cmd.do()
+        Returns:
+
+            The return value of cmd.do()
```

Comment:
I think you can just remove the information about the return value here as it isn't interesting.